src/libusbmuxd.c

```diff
@@ -731,7 +739,7 @@ static plist_t create_plist_message(const char* message_type)
 	if (bundle_id) {
 		plist_dict_set_item(plist, "BundleID", plist_new_string(bundle_id));
 	}
-	plist_dict_set_item(plist, "ClientVersionString", plist_new_string(PLIST_CLIENT_VERSION_STRING));
+	plist_dict_set_item(plist, "ClientVersionString", plist_new_string(libusbmuxd_version()));
```

Comment:
This is not exactly the same. PACKAGE_STRING contains the package name and version, like `libusbmuxd 2.1.0` while libusbmuxd_version() only returns the version part which would be `2.1.0`.

Reply:
Good catch, fixed it!